build/ci/job-template.yml

```diff
@@ -148,16 +148,6 @@ jobs:
     - script: $(dotnetPath) msbuild -restore build/Codecoverage.proj
       displayName: Upload coverage to codecov.io
       condition: and(succeeded(), eq(${{ parameters.codeCoverage }}, True))
-    - task: PublishTestResults@2
```

Comment:
Don't we want this? I don't see the `Test` tab on the CI build in this PR:

![image](https://user-images.githubusercontent.com/8291187/101528202-9aad0b00-3954-11eb-8c93-0760102d678c.png)

Publishing the test results allows Azure DevOps to easily show test failures:

![image](https://user-images.githubusercontent.com/8291187/101528354-d47e1180-3954-11eb-928b-8789b91d02d0.png)

Reply:
after migrate to arcade we are not generating any trx test results so this step is not necessary any more. we have a stage (the next one) that upload test results (html format) if test fails.

---
In reply to: [538723243](https://github.com/dotnet/machinelearning/pull/5541#discussion_r538723243) [](ancestors = 538723243)

Reply:
I think Eric's point maybe that Azure has the capability to show test failures right away without having to scan through the build logs. We aren't generating trx test results, but maybe Arcade has a different mechanism to enable this. Can you please look into it?

---
In reply to: [538732212](https://github.com/dotnet/machinelearning/pull/5541#discussion_r538732212) [](ancestors = 538732212,538723243)

Reply:
Sure, I will take a look, that can be done in separate PR.

---
In reply to: [538734618](https://github.com/dotnet/machinelearning/pull/5541#discussion_r538734618) [](ancestors = 538734618,538732212,538723243)

Reply:
Sounds good.

---
In reply to: [538752000](https://github.com/dotnet/machinelearning/pull/5541#discussion_r538752000) [](ancestors = 538752000,538734618,538732212,538723243)